src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -72,21 +74,26 @@ public CommandResult execute(Model model) throws CommandException {
 
     /**
      * Creates and returns a {@code Person} with the details of {@code personToEdit}
-     * edited with {@code editPersonDescriptor}.
+     * with added tags from {@code editPersonDescriptor}.
      */
     private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
-        assert personToEdit != null;
+        requireNonNull(personToEdit);
 
-        Name updatedName = personToEdit.getName();
-        Phone updatedPhone = personToEdit.getPhone();
-        Email updatedEmail = personToEdit.getEmail();
-        Address updatedAddress = personToEdit.getAddress();
-        Birthday updatedBirthday = personToEdit.getBirthday().orElse(null);
+        Name unchangedName = personToEdit.getName();
+        Phone unchangedPhone = personToEdit.getPhone();
+        Email unchangedEmail = personToEdit.getEmail();
+        Address unchangedAddress = personToEdit.getAddress();
+        Birthday unchangedBirthday = personToEdit.getBirthday().orElse(null);
 
+        Set<Tag> existingTags = personToEdit.getTags();
+        Set<Tag> addedTags = editPersonDescriptor.getTags().orElse(new HashSet<Tag>());
         Set<Tag> updatedTags = new HashSet<Tag>();
-        updatedTags.addAll(personToEdit.getTags());
-        updatedTags.addAll(editPersonDescriptor.getTags().orElse(new HashSet<Tag>()));
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags, updatedBirthday);
+        updatedTags.addAll(existingTags);
+        updatedTags.addAll(addedTags);
+
+        return new Person(unchangedName, unchangedPhone, unchangedEmail, unchangedAddress,
+                updatedTags,
+                unchangedBirthday);
```

Comment:
I think can be the same line as `updatedTags`

Reply:
purpose is to put the updated tags on its own line to distinguish from the unchanged items